src/btouch.cs

```diff
@@ -224,8 +236,8 @@ static int Main2 (string [] args)
 					linkwith.Add (id);
 				}
 			},
-			{ "unified-full-profile", "Launches compiler pointing to XM Full Profile", l => { /* no-op*/ }, true },
```

Comment:
Love the idea, but I chose a different name here:

https://github.com/xamarin/xamarin-macios/blob/master/tools/mmp/driver.cs#L362

I kinda like yours better. I can PR after this goes in swapping the name unless you want to add it here.